.dev/tests/cypress/helpers.js

```diff
@@ -131,11 +167,13 @@ export function savePage() {
  * Check the page for block errors
  *
  * @param string blockID Optional ID to check for in the DOM.
- *               Note: If no blockID is specified, getBlockSlug() attempts to
- *               retreive the block from the spec file.
- *               eg: accordion => div[data-type="coblocks/accordion"]
+ * Note: If no blockID is specified, getBlockSlug() attempts to
+ * retreive the block from the spec file.
+ * eg: accordion => div[data-type="coblocks/accordion"]
+ * @param blockID
+ * @param slug
  */
-export function checkForBlockErrors( blockID = '' ) {
+export function checkForBlockErrors( blockID = '', slug = 'coblocks' ) {
```

Comment:
Again, let's change `blockId` to `blockName` and pass the full namespaced block name to this function. Remove the new parameter.